src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,8 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        StringBuilder reverseText = new StringBuilder(text = text.replaceAll("[^a-zA-Z0-9]",""));
```

Comment:
@Valdes2 try to not initialize two variables in one statement
```suggestion
        text = text.replaceAll("[^a-zA-Z0-9]","")
        StringBuilder reversedText = new StringBuilder(text).reverse();
```

Reply:
fixed